packages/material-ui-lab/src/TimelineOppositeContent/TimelineOppositeContent.js

```diff
@@ -59,8 +96,12 @@ TimelineOppositeContent.propTypes /* remove-proptypes */ = {
    * @ignore
    */
   className: PropTypes.string,
+  /**
+   * The system prop that allows defining system overrides as well as additional CSS styles.
+   */
+  sx: PropTypes.any,
 };
 
 TimelineOppositeContent.muiName = 'TimelineOppositeContent';
```

Comment:
This gave me insane headaches. At first I did not understand why the component did not work but when I investigated I realized that in `TimelineItem` there is a logic to add classes if there is `TimelineOpppositeContent` as a child.

I wonder if all components should have this or only if necessary?

Reply:
Only if necessary, we recently added types for it:

https://github.com/mui-org/material-ui/blob/a14696593902fd422b8f759d45b1b7ee409223f1/packages/material-ui-lab/src/TimelineOppositeContent/TimelineOppositeContent.d.ts#L36